ext/libxml/libxml.c

```diff
@@ -1385,6 +1409,15 @@ PHP_LIBXML_API void php_libxml_node_free_resource(xmlNodePtr node)
 		case XML_DOCUMENT_NODE:
 		case XML_HTML_DOCUMENT_NODE:
 			break;
+		case XML_ENTITY_REF_NODE:
+			/* Entity reference nodes are special: their children point to entity declarations,
+			 * but they don't own the declarations and therefore shouldn't free the children.
+			 * Moreover, there can be N>1 reference nodes for a single entity declarations. */
```

Comment:
The ``N>1`` could potentially cause confusing here? Not sure what better wording is possible tho.

Reply:
Fix:

```suggestion
			 * Moreover, there can be more than one reference node for a single entity declarations. */
```

Reply:
The suggestion looks good, thank you!